digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/RoadLinkDAO.scala

```diff
@@ -365,6 +382,7 @@ class RoadLinkDAO {
       sql"""select linkid
           from kgv_roadlink
           where #$polygonFilter
+          and expired_date isnull
```

Comment:
Tuli mieleen, että nyt kun tarkastetaan ainoastaan että onko expired_date null vai ei, niin pitäisikö lisätä kgv_roadlink tauluun constraint expired_date kentälle että se ei voi olla tulevaisuudessa (expired_date <= current_timestamp).

Reply:
Voin lisätä. Ilmeisesti me halutaan, että päivä ei saa olla tulevaisuudessa?

Reply:
No näin mä ajattelisin. Jos päivä olisi tulevaisuudessa niin silloinhan linkin pitäisi palautua kyselyissä siihen asti että kyseinen päivä tulee ja pelkkä is null tarkastus ei toimisi. Ajatuksena oli että muutos lambda merkkaa aina vanhentuneet linkit, joten päivä on yleensä kuluva päivä.